### PR TITLE
Bump Spinnaker to 1.14.11 and Halyard to 1.22.1.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

File: scripts/install/setup_properties.sh [hunk 1]
```diff
@@ -88,8 +88,8 @@ else
 export PROJECT_ID=$PROJECT_ID
 export DEPLOYMENT_NAME=${DEPLOYMENT_NAME:-$NEW_DEPLOYMENT_NAME}
 
-export SPINNAKER_VERSION=1.14.8
-export HALYARD_VERSION=1.21.2
+export SPINNAKER_VERSION=1.14.11
+export HALYARD_VERSION=1.22.0
```
Comment (plumpy) [inline scripts/install/setup_properties.sh]: 1.22.1! Released yesterday!

Comment (duftler) [reply scripts/install/setup_properties.sh]: Updated.

**Comment by demobox** [comment]
Any breaking changes that we should be aware of?

**Comment by duftler** [comment]
Plumpy fixed the one potential breaking change by making halyard tolerant
of both styles of oauth scopes in the hal config.

**Pull request merged**